### PR TITLE
DEN-2525: Remove attributes if value is null

### PR DESCRIPTION
**Pull request by migo315**

### CODE REVIEW

File: src/ActionBuilder/Product/SetAttributes.php [hunk 1]
```diff
@@ -67,18 +67,22 @@ public function createUpdateActions(
...
-
-            // ENUM and LENUM Attributes could contain an empty label (=null) because we only set value and no label.
-            // Therefor, ODM detects a change for the label, which is uninteresting for us.
-            // We don't want any action if the only changed field is the 'null label'.
-            if (array_key_exists('value', $attr) && is_array($attr['value']) && count($attr['value']) === 1) {
-                if (array_key_exists('label', $attr['value']) && $attr['value']['label'] === null) {
-                    continue;
-                }
-            }
+            $attributeName = $attr['name'] ?? $oldAttrs[$attrIndex]['name'];
```
Comment (avareldev) [inline src/ActionBuilder/Product/SetAttributes.php:70]: Why did you move this here?

**Review by avareldev — APPROVED** [review]

File: src/ActionBuilder/Product/SetAttributes.php (hunk 1 above)
Comment (migo315) [reply src/ActionBuilder/Product/SetAttributes.php:70]: Because I don't think it is nice to pass three variable to a method just to generate a name.
